rules/react.js

```diff
@@ -0,0 +1,69 @@
+module.exports = {
+  env: {
+    browser: true
+  },
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true
+    }
+  },
+  plugins: ['react'],
+  settings: {
+    react: {
+      pragma: 'React',
+      version: '16.4'
```

Comment:
```suggestion
      version: '16.6'
```

I've a PR that bumps react to 16.6, would save the go-around of changing this here too. Up to you.